utility/cli/options.cpp

```diff
@@ -170,7 +170,7 @@ namespace beam
         const char* LOG_ERROR = "error";
         const char* LOG_WARNING = "warning";
         const char* LOG_INFO = "info";
-        const char* LOG_DEBUG = "debug";
+        const char* BEAM_LOG_DEBUG = "debug";
```

Comment:
I don't think that this should be changed

Reply:
reverted